File: QuartzNetWebConsole/Models/TriggerWithState.cs
using Quartz;

namespace QuartzNetWebConsole.Models {
    public struct TriggerWithState {
        private readonly Trigger trigger;
        private readonly TriggerState state;

        public TriggerWithState(Trigger trigger, TriggerState state) {
            this.trigger = trigger;
            this.state = state;
        }

        public Trigger Trigger {
            get { return trigger; }
        }

        public TriggerState State {
            get { return state; }
        }

        public bool IsPaused {
            get { return state == TriggerState.Paused; }
        }
    }
}

File: QuartzNetWebConsole.Views/TriggerWithState.vb
Imports Quartz

Public Class TriggerWithState
    Public ReadOnly Trigger As ITrigger
    Public ReadOnly State As TriggerState

    Public Sub New(trigger As ITrigger, state As TriggerState)
        Me.Trigger = trigger
        Me.State = state
    End Sub

    Public ReadOnly Property IsPaused As Boolean
        Get
            Return State = TriggerState.Paused
        End Get
    End Property
End Class
